lib/active_model_serializers/model.rb

```diff
@@ -30,6 +30,8 @@ def updated_at
     end
 
     def read_attribute_for_serialization(key)
+      return send(key) if respond_to?(key)
```

Comment:
that's true. I think that's just another point for the current AMS::Model being broken. :-(

Do you have a PR for your module idea?